x-pack/legacy/plugins/siem/public/components/timeline/properties/helpers.tsx

```diff
@@ -111,6 +113,47 @@ export const Name = React.memo<NameProps>(({ timelineId, title, updateTitle }) =
 ));
 Name.displayName = 'Name';
 
+interface NewCaseProps {
+  onClosePopover: () => void;
+  timelineId: string;
+  timelineTitle: string;
+}
+
+export const NewCase = React.memo<NewCaseProps>(({ onClosePopover, timelineId, timelineTitle }) => {
+  const [createCase, setCreateCase] = useState(false);
+  const handleClick = useCallback(() => {
+    setCreateCase(true);
+    onClosePopover();
+  }, [onClosePopover]);
+  if (createCase) {
+    return (
+      <Redirect
```

Comment:
maybe instead of the `Redirect` we could use https://reacttraining.com/react-router/web/api/Hooks/usehistory ?

Reply:
will try it out

Reply:
I can't get this to work with the state. tried it a few ways, look together in the tomorrow times?